src/libcoretest/fmt/num.rs

```diff
@@ -156,96 +156,98 @@ fn test_format_int_twos_complement() {
 
 #[test]
 fn test_format_radix() {
-    assert!(format!("{:04}", radix(3i, 2)) == "0011");
-    assert!(format!("{}", radix(55i, 36)) == "1j");
+    assert!(format!("{:04}", radix(3, 2)) == "0011");
+    assert!(format!("{}", radix(55, 36)) == "1j");
 }
 
 #[test]
 #[should_fail]
 fn test_radix_base_too_large() {
-    let _ = radix(55i, 37);
+    let _ = radix(55, 37);
 }
 
-mod uint {
+mod u32 {
     use test::Bencher;
     use core::fmt::radix;
     use std::rand::{weak_rng, Rng};
+    use std::io::util::NullWriter;
 
     #[bench]
     fn format_bin(b: &mut Bencher) {
         let mut rng = weak_rng();
-        b.iter(|| { format!("{:b}", rng.gen::<uint>()); })
+        b.iter(|| { let _ = write!(&mut NullWriter, "{:b}", rng.gen::<u32>()); })
```

Comment:
Is there a reason to feed this into a `let` and not just return it into the closure?